SourceEditorCompose/src/main/res/xml/data_extraction_rules.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="utf-8"?><!--
```

Comment:
Same here (we probably don't need this file nor the `xml` folder).

Reply:
@UnmeshPadhye would you please remove this file as Cesar mentioned?